riscv-rt/src/interrupts.rs

```diff
@@ -14,64 +14,7 @@
 //! code to adapt for the target needs. In this case, you may need to opt out this module.
```

Comment:
Could you add a few lines of documentation for vectored mode about the `RISCV_MTVEC_ALIGN` environment variable and why it is necessary? You should also mention that, in most of the cases, this environment variable must be handled by the PAC crate, not users directly

Reply:
I'm not good at documentation, so I attempted something. Feel free to suggest an alternative.